Validation/coordinates.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+function validateFloat($longitude, $latitude) : string
```

Comment:
function validateFloat($value)
{
    if (!filter_var($longitude, FILTER_VALIDATE_FLOAT) {
		return "ERROR: Type of value must be float!";
    }
		return "Type of value is correct! Thanks!";
}